pystiche_papers/johnson_alahi_li_2016/_modules.py

```diff
@@ -204,7 +204,9 @@ def decoder(
 ) -> pystiche.SequentialModule:
     def get_value_range_delimiter() -> nn.Module:
         if impl_params:
-
+            # https://github.com/pmeier/fast-neural-style/blob/813c83441953ead2adb3f65f4cc2d5599d735fa7/train.lua#L25
+            # https://github.com/pmeier/fast-neural-style/blob/813c83441953ead2adb3f65f4cc2d5599d735fa7/fast_neural_style/models.lua#L137-L138
+            # A tanh with a constant factor of 150 is used instead of the (tanh(x) + 1) / 2 in the paper.
```

Comment:
Could you have a look at the paper again? I don't recall them mentioning `tanh`.

Reply:
In the paper it is described like this: ([paper](https://cs.stanford.edu/people/jcjohns/papers/eccv16/JohnsonECCV16.pdf) first section chapter 3.1)
> ..the exception of the output layer, which instead uses a scaled tanh to ensure that the output has pixels in the range [0,255].